### ISSUE TITLE
How is the dataset splited into training set and validation set

### ISSUE THREAD

**Issue by CubicSalt** [miccunifi/FRED#1]
Very significant work. 

I noticed that the paper mentioned two ways of splitting the dataset, but I'm wondering how they (231 zips) were divided.

Thank you.

**Comment by ChunyangYuan** [comment]
Very significant work.

I noticed that the paper mentioned two ways of splitting the dataset, but I'm wondering how they (231 zips) were divided.

Thank you.
